### ISSUE TITLE
String cannot be cast to AccessibilityRole

### ISSUE THREAD

**Issue by nayan27** [react-native-community/react-native-tab-view#734]
### Current behaviour

Using the sample code provided in the README's Quick Start section gives me this:

### Code sample

The code is the exact same as provided in README. Using the component as the initial route in a react-navigation switch navigator.

### Screenshots (if applicable)

![Screenshot_20190401-153746](https://user-images.githubusercontent.com/6426330/55354854-c1832080-5494-11e9-9046-5277bbd6cc42.png)

### Your Environment

| software                     | version
| ---------------------------- | -------
| ios or android               | Android
| react-native                 | ^0.57.3
| react-native-tab-view        | ^2.0.1
| react-native-gesture-handler | ^1.1.0
| react-native-reanimated      | ^1.0.0-alpha.12
| node                         | V8.12.0
| npm or yarn                  | 6.4.1

Anyone encountered this or have an idea on a workaround ?

Thanks!

**Comment by satya164** [comment]
Upgrade your React Native version. It's a bug in React Native https://github.com/facebook/react-native/issues/21754

**Comment by nayan27** [comment]
Thanks ! upgrading fixed the problem!
